common/config.gni

```diff
@@ -75,7 +75,7 @@ if (_platform_name == "mac") {
   _platform_name = "windows"
 }
 
-full_platform_name = "$_platform_name-$host_cpu"
+full_platform_name = "$_platform_name-$target_cpu"
```

Comment:
Mixing the host os with the target cpu doesn't make sense. Only host_os/host_cpu or target_os/target_cpu makes sense.

Reply:
Created an additional variable using target_os and target_cpu which is required for the output name of the dart-sdk and left this one intact.